openedx/core/djangoapps/credentials/tests/mixins.py

```diff
@@ -33,103 +34,70 @@ class CredentialsDataMixin(object):
     CREDENTIALS_API_RESPONSE = {
         "next": None,
         "results": [
-            {
-                "id": 1,
-                "username": "test",
-                "credential": {
-                    "credential_id": 1,
-                    "program_id": 1
-                },
-                "status": "awarded",
-                "uuid": "dummy-uuid-1",
-                "certificate_url": "http://credentials.edx.org/credentials/dummy-uuid-1/"
-            },
-            {
-                "id": 2,
-                "username": "test",
-                "credential": {
-                    "credential_id": 2,
-                    "program_id": 2
-                },
-                "status": "awarded",
-                "uuid": "dummy-uuid-2",
-                "certificate_url": "http://credentials.edx.org/credentials/dummy-uuid-2/"
-            },
-            {
-                "id": 3,
-                "username": "test",
-                "credential": {
-                    "credential_id": 3,
-                    "program_id": 3
-                },
-                "status": "revoked",
-                "uuid": "dummy-uuid-3",
-                "certificate_url": "http://credentials.edx.org/credentials/dummy-uuid-3/"
-            },
-            {
-                "id": 4,
-                "username": "test",
-                "credential": {
-                    "course_id": "edx/test01/2015",
-                    "credential_id": 4,
-                    "certificate_type": "honor"
-                },
-                "status": "awarded",
-                "uuid": "dummy-uuid-4",
-                "certificate_url": "http://credentials.edx.org/credentials/dummy-uuid-4/"
-            },
-            {
-                "id": 5,
-                "username": "test",
-                "credential": {
-                    "course_id": "edx/test02/2015",
-                    "credential_id": 5,
-                    "certificate_type": "verified"
-                },
-                "status": "awarded",
-                "uuid": "dummy-uuid-5",
-                "certificate_url": "http://credentials.edx.org/credentials/dummy-uuid-5/"
-            },
-            {
-                "id": 6,
-                "username": "test",
-                "credential": {
-                    "course_id": "edx/test03/2015",
-                    "credential_id": 6,
-                    "certificate_type": "honor"
-                },
-                "status": "revoked",
-                "uuid": "dummy-uuid-6",
-                "certificate_url": "http://credentials.edx.org/credentials/dummy-uuid-6/"
-            }
+            factories.UserCredentials(
+                id=1,
+                username='test',
+                credential=factories.ProgramCredential(
+                    program_id=1
```

Comment:
Since you're using a Sequence for this attribute, you shouldn't need to provide it manually like this anywhere. I think just `factories.ProgramCredential()` should be enough.